checker/cost.go

```diff
@@ -599,3 +608,20 @@ func (c *coster) newAstNode(e *exprpb.Expr) *astNode {
 	}
 	return &astNode{path: path, t: c.getType(e), expr: e, derivedSize: derivedSize}
 }
+
+// isScalar returns true if the given type is known to be of a constant size at
```

Comment:
Do we need to care about the scalar wrapper types?

Reply:
This is a backport of a fix currently in v0.10.2, but not on v0.10.3 ... Happy to revisit the question.